### ISSUE TITLE
RE: Private Bytes on Azure Web App grows until web app fails to respond to request

### ISSUE THREAD

**Issue by riiight** [dotnet/aspnetcore#29274]
This issue was created because [this issue](https://github.com/dotnet/aspnetcore/issues/3562) was answered and closed, but the actual [answer](https://github.com/dotnet/aspnetcore/issues/3562#issuecomment-427195118) wasn't included but instead had a [link that now redirects to something else](https://docs.microsoft.com/en-us/ef/core/miscellaneous/logging).

I believe I may be experiencing the [same issue](https://github.com/dotnet/aspnetcore/issues/3562) as @foconnor-DS and would just like to hear what the solution is (i.e. what was the big, red warning?). @sebastienros provided the answer itself (i.e. the link), so if either of you can provide the solution, I'd very much appreciate it.

Apologies if this could have been requested in a better way.

**Comment by riiight** [comment]
Created in error.

**Comment by sebastienros** [comment]
Sorry, kind of unfortunate. Did some git history digging and found these:

```
> [!WARNING]
> In EF Core 2.1, It is very important that applications do not create a new LoggerFactory instance for each DbContext instance. Doing so will result in a memory leak and poor performance. This has been fixed in EF Core 3.0 and above.
```

And in a sample:

```c#
    public class BloggingContext : DbContext
    {
        #region DefineLoggerFactory
        public static readonly ILoggerFactory MyLoggerFactory
            = LoggerFactory.Create(builder => { builder.AddConsole(); });
        #endregion

        public DbSet<Blog> Blogs { get; set; }

        #region RegisterLoggerFactory
        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
            => optionsBuilder
                .UseLoggerFactory(MyLoggerFactory) // Warning: Do not create a new ILoggerFactory instance each time
                .UseSqlServer(@"Server=(localdb)\mssqllocaldb;Database=EFLogging;ConnectRetryCount=0");
        #endregion
    }
```

I assume that's what I was referring to.
